test/relations.test.js

```diff
@@ -267,6 +267,58 @@ describe('relations', function() {
         }
       });
 
+      it('should fetch all scoped instances with getAsync with callback and options', function(done) {
+        Book.create(function(err, book) {
+          book.chapters.create({name: 'a'}, function() {
+            book.chapters.create({name: 'z'}, function() {
+              book.chapters.create({name: 'c'}, function() {
+                verify(book);
+              });
+            });
+          });
+        });
+        function verify(book) {
+          book.chapters({}, {options: 'options'}, function(err, ch) {
+            should.not.exist(err);
```

Comment:
```js
if (err) return done(err);
```

Your style hides stack trace from the original error and reports a stack trace pointing only to this assertion, which is not very helpful.